index.html

```diff
@@ -1472,37 +1472,32 @@ <h2>
         subscription</a>.
       </p>
       <p>
-        The <dfn>getKey()</dfn> method retrieves keying material that can be used for encrypting
-        and authenticating messages. When {{PushSubscription/getKey()}} is invoked the following
-        process is followed:
+        The <dfn>getKey</dfn><code>(|name:PushEncryptionKeyName|)</code> method retrieves keying
+        material that can be used for encrypting and authenticating messages. The method steps are:
       </p>
       <ol>
-        <li>Find the internal slot corresponding to the key named by the `name` argument.
-        </li>
-        <li>If a slot was not found, return `null`.
-        </li>
-        <li>Initialize a variable |key| with a newly instantiated {{ArrayBuffer}} instance.
+        <li>Let |key| be a newly instantiated {{ArrayBuffer}} instance.
         </li>
-        <li>If the internal slot contains an asymmetric key pair, set the contents of |key| to the
-        serialized value of the public key from the key pair. This uses the serialization format
-        described in the specification that defines the name. For example, [[RFC8291]] specifies
-        that the {{PushEncryptionKeyName/"p256dh"}} public key is encoded using the uncompressed
-        format defined in [[ANSI-X9-62]] Annex A (that is, a 65 octet sequence that starts with a
-        0x04 octet).
+        <li>If |name| is {{PushEncryptionKeyName/"p256dh"}}:
+          <ol>
+            <li>Set the contents of |key| to the serialized value of the public key from
+            [=this=]'s [=P-256 ECDH key pair=], using the uncompressed format defined in
+            [[ANSI-X9-62]] Annex A (that is, a 65 octet sequence that starts with a 0x04 octet), as
+            [[RFC8291]] specifies.
+            </li>
+          </ol>
         </li>
-        <li>Otherwise, if the internal slot contains a symmetric key, set the contents of |key| to
-        a copy of the value from the internal slot. For example, the `auth` parameter contains an
-        octet sequence used by the <a>user agent</a> to authenticate messages sent by an
-        <a>application server</a>.
+        <li>Otherwise:
+          <ol>
+            <li>[=/Assert=]: |name| is {{PushEncryptionKeyName/"auth"}}</li>
```

Comment:
I think you can just do: 

```suggestion
            <li>Assert: |name| is {{PushEncryptionKeyName/"auth"}}</li>
```

And if that should link to WebIDL's assert, we should probably fix that in ReSpec (can't remember if it links already or not).

Reply:
No, doesn't seem to work. Just a plaintext.